web/src/components/tasks/business-formation/business/PracticesLaw.tsx

```diff
@@ -0,0 +1,72 @@
+import { WithErrorBar } from "@/components/WithErrorBar";
+import { BusinessFormationContext } from "@/contexts/businessFormationContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { FormationWillPracticeLaw } from "@businessnjgovnavigator/shared/formationData";
+import { FormControl, FormControlLabel, FormHelperText, FormLabel, Radio, RadioGroup } from "@mui/material";
+import { ReactElement, useContext } from "react";
+
+interface Props {
+  hasError: boolean;
+}
+
+export const PracticesLaw = (props: Props): ReactElement => {
+  const { hasError } = props;
+  const { Config } = useConfig();
+  const { state, setFormationFormData } = useContext(BusinessFormationContext);
+
+  const onChange = (value: FormationWillPracticeLaw): void => {
+    setFormationFormData((previousState) => {
+      return {
+        ...previousState,
+        willPracticeLaw: value,
+      };
+    });
+  };
+
+  return (
+    <WithErrorBar hasError={hasError} type="ALWAYS">
+      <FormControl variant="outlined" fullWidth className="padding-bottom-2">
+        <RadioGroup
+          aria-label={"Foreign address type"}
```

Comment:
copy paste error? Usually we use `camelCaseToSentenceCase` of the field name for aria labels